2024/02/hms5232/.idea/vcs.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
While I personally won't upload IDE specific files to a public repo, it is fine as it is not big.

Reply:
OK, i will ignore those from git in the future and delete those in next PR (if i can resolve the problem)